addOns/bruteforce/src/main/java/org/zaproxy/zap/extension/bruteforce/BruteForce.java

```diff
@@ -108,6 +109,12 @@ public BruteForce(
 
         if (bruteForceParam.isBrowseFiles()) {
             extensions = bruteForceParam.getFileExtensionsList();
+            if (bruteForceParam.isBrowseFilesWithoutExtension()) {
+                extensions.add(ExtToCheck.BLANK_EXT);
```

Comment:
The `extensions` might be an unmodifiable collection (when Browse Files is enabled and no extension is provided).

Reply:
This one was not addressed yet, it will still lead to `UnsupportedOperationException`.

Reply:
Sry I didn't understood what was needed and I did other change instead.

Reply:
No worries.